aws/resource_aws_route53_record_test.go

```diff
@@ -1394,7 +1470,7 @@ resource "aws_route53_record" "alias" {
 
 resource "aws_elb" "main" {
   name = "foobar-terraform-elb-%s"
-  availability_zones = ["us-west-2a"]
+  availability_zones = slice(data.aws_availability_zones.available.names, 0, 1)
```

Comment:
These handful of tests were hardcoded to us-west-2 and can only pass in that region. This just updates them to work in other regions so I could post the full acceptance test output.